test/append-body.test.js

```diff
@@ -140,6 +140,73 @@ test('append to body option and multiple files', t => {
   })
 })
 
+test('append to body option and multiple files in one field', t => {
```

Comment:
```suggestion
test('addToBody option and multiple files in one field', t => {
```

Can you also update this in the other tests? Thanks.

Reply:
@mcollina you mean change that in all tests, even those that wasn't changed, right?

Reply:
yes

Reply:
done